src/components/content-container/styles.css

```diff
@@ -0,0 +1,23 @@
+.content-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  overflow: hidden;
+  width: 400px;
+  background-color: #fff;
+  height: 100%;
+  border-radius: 20px;
+  padding: 20px;
+  justify-content: space-between;
+}
+
+@media screen and (max-width: 920px) {
+  .content-container {
+    width: 80%;
+    background: url('../../pages/shopping-add-image.svg') no-repeat;
```

Comment:
Wow! An SVG as background. ⭐⭐ 😮

Reply:
I loved it too! It was Mai's idea! @mailliwmogollon 😍